.github/workflows/tests-linux.yml

```diff
@@ -68,10 +69,6 @@ jobs:
 
   mssql:
     runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        mssql-version: ["^9.1.1", "^10.0.1", "^11.0.1"]
```

Comment:
I went looking through git blame to figure out why we were testing all these `mssql` client versions in the first place (added in #10933).

Turns out there wasn't really any rationale for it. We don't test any other client libraries to this extent, and the only breaking changes in these `mssql` releases were just dropping old Node versions. So it seems fine to stop doing this. :+1:

Reply:
I agree. Different versions of the clients rarely make a difference. I think we get a bigger difference if we have different DB versions, since they have more changes.

I'm hoping in the next major release we'll have a single major version of the clients as minimum dependency and that we'll bump some docker image versions as well (especially mysql which I can't run locally through docker since it's too old).